docs/billing-and-plans/cancel-active-paid-plan.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I suggest rename the files to emphasize the word `subscription`.

cancel subscription
upgrade subscripton
etc

Reply:
Solved in d1c5360